artifact_requirements.txt

```diff
@@ -11,7 +11,11 @@ pandas
 pyyaml
 scipy
 tables
-vivarium_inputs[data]>=5.0.0
+vivarium_inputs>=5.0.0
+# Note inputs is not install with [data] extra because we might need to specify the branch
```

Comment:
This shouldn't trigger the grep, since I have a space instead of an underscore?
```suggestion
# Note: vivarium inputs is not installed with [data] extra because we might need to specify the branch
```

Reply:
No its just getting any line that has 'vivarium' in it.

Reply:
Made it viv_in